ecosystem/sep-0010.md

```diff
@@ -18,12 +18,18 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
+It involves three parties:
+- A home domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` value.<sup>[1](#note1)</sup>
+- A web auth endpoint providing the GET and POST operations discussed in this document.
+- A client.
+
 The authentication flow is as follows:
 
-1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
+1. The client obtains the `WEB_AUTH_ENDPOINT` for the service by getting the [SEP-1 stellar.toml](sep-0001.md) from the services domain, referred to as the home domain. The home domain hosts the stellar.toml.
+1. The client obtains a unique [`challenge`](#challenge) from the `WEB_AUTH_ENDPOINT`, which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
 1. The client verifies that the transaction is signed by the `SIGNING_KEY` specified by the requested service's [SEP-1 stellar.toml](sep-0001.md).
-1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client ignores the home domain included.
+1. The client verifies that the transaction has a single Manage Data operation with its source account set to the user's account and value set to a nonce value. The client verifies the home domain included is the domain from the first step.
```

Comment:
> the first step

are you referring to this?

> 1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction

This makes it seem like the domain should be the WEB_AUTH_ENDPOINT, not the SEP-1 stellar.toml URL's hostname.

Reply:
The first step was changed to be: 
>1. The client obtains the `WEB_AUTH_ENDPOINT` for the service by getting the [SEP-1 stellar.toml](sep-0001.md) from **the services domain, referred to as the home domain**. The home domain hosts the stellar.toml.

Maybe I can make this clearer. Suggestions?

Reply:
Oh no I was mistaken and thought the line you were removing was still present. This looks good.

Reply:
I think it could be clearer. It's confusing the first step talks about the endpoint first rather than the stellar.toml. I've made a change in 0c4f2697d50b84e5a5a577d67e46ac5b5959b097 to make it clearer.

Reply:
Now that there are multiple lists I want to reiterate that `the first step` should be replaced with a better reference.

Reply:
💯 How is this? eb00ba8